src/meltano/cli/cli.py

```diff
@@ -43,7 +43,8 @@ def main(self, *args, **kwargs) -> t.NoReturn:
             args: Positional arguments for the Click group.
             kwargs: Keyword arguments for the Click group.
         """
-        return super().main(*args, windows_expand_args=False, **kwargs)
+        # TODO: Fix "Return statement in function which does not return"?
+        return super().main(*args, windows_expand_args=False, **kwargs)  # type: ignore[misc]
```

Comment:
Why are we returning anything from this function? If the return value is significant, why not change the type signature?

Reply:
Yeah, I don't know why that didn't occur to me 😅

1a9fba1b7